docs/user/quickstart.rst

```diff
@@ -153,9 +153,9 @@ There's also a builtin JSON decoder, in case you're dealing with JSON data::
 
 In case the JSON decoding fails, ``r.json()`` raises an exception. For example, if
 the response gets a 204 (No Content), or if the response contains invalid JSON,
-attempting ``r.json()`` raises ``simplejson.JSONDecodeError`` if simplejson is
-installed or raises ``ValueError: No JSON object could be decoded`` on Python 2 or
-``json.JSONDecodeError`` on Python 3.
+attempting ``r.json()`` raises ``requests.JSONDecodeError``. This is new for the
```

Comment:
We probably don't want to add all the information in the last sentence as it will become stale quickly. Let's simplfy it to something closer to:

> This wrapper exception provides interoperability for multiple exceptions that may be thrown by different python versions and json serialization libraries.

Reply:
Ok. I'll change that.